featurebyte/routes/use_case/controller.py

```diff
@@ -70,14 +70,22 @@ async def create_use_case(self, data: UseCaseCreate) -> UseCaseModel:
         UseCaseModel
         """
         # validate both target and context exists
-        target = await self.target_service.get_document(document_id=data.target_id)
         context = await self.context_service.get_document(document_id=data.context_id)
 
+        target_namespace = None
+        if data.target_id:
```

Comment:
if we make namespace id non optional, can we simplify this to always look for the entity ids from the namespace?